src/client/common/components/base-network-view/base-network-view.js

```diff
@@ -218,7 +225,8 @@ class BaseNetworkView extends React.Component {
         ]
       ),
       h('div', {
-        className: classNames('sidebar-menu', { 'sidebar-menu-open': this.state.open })
+        className: classNames('sidebar-menu',{'sidebar-menu-open': this.state.open }),
+        style: { width: menuWidth?`${menuWidth}%`:'' }
```

Comment:
What happens to the visuals of the sidebar if width is ''?

Reply:
The width property gets ignored and the width from the class sidebar-menu-open is used